buildSrc/src/main/java/com/uber/okbuck/core/model/jvm/JvmTarget.java

```diff
@@ -291,13 +297,26 @@ public static String javaVersion(JavaVersion version) {
 
       for (SubpluginOption option : options) {
         optionBuilder.add("-P");
-        optionBuilder.add(
-            "plugin:org.jetbrains.kotlin.allopen:" + option.getKey() + "=" + option.getValue());
+          optionBuilder.add(
+              "plugin:org.jetbrains.kotlin.allopen:" + option.getKey() + "=" + option.getValue());
+        }
```

Comment:
seems like your git hook for GJF is not setup -- https://github.com/uber/okbuck/blob/master/README.md#code-style

Reply:
I think it is? This line was too long before o_O